src/udev/udevd.c

```diff
@@ -983,7 +983,7 @@ static int event_queue_start(Manager *manager) {
                                                  event->seqnum,
                                                  strna(device_action_to_string(a)));
 
-                        event_free(event);
+                        event_enter_failed(event);
```

Comment:
hmm, under which conditions will event_is_blocked() actually fail? 

wouldn't a better strategy be to warn if the function fails, and then just run the event? i.e. to treat return value < 0 like r == 0 effectively, besides the additional logging?